src/sass/base/_typography.scss

```diff
@@ -1,3 +1,3 @@
-/**
- * Basic typography style for copy text
- */
\ No newline at end of file
+// -----------------------------------------------------------------------------
+// This file contains default typographic styles copy text.
```

Comment:
Missing "for"
...typographic stiles _for_ copy text.

Reply:
Absolutely :)